docs/freqtrade-101.md

```diff
@@ -0,0 +1,106 @@
+# Freqtrade Strategies 101: A Quick Start for Strategy Development
+
+For the purposes of this quick start, we are assuming you are familiar with the basics of trading.
+
+## Required Knowledge
+
+A strategy in Freqtrade is a Python class that defines the logic for buying and selling cryptocurrency `assets`.
+
+Assets are defined as `pairs`, which represent the `coin` and the `stake`. The coin is the asset you are trading using another currency as the stake.
```

Comment:
we might want to reference freqtrade basics here ? it covers this topic in more depth ... ? (unless we want to explicitly keep it concise and "non-referencing" - so readers can stay on this page).